ta/bonnie/bonnie.c

```diff
@@ -0,0 +1,79 @@
+/*
```

Comment:
You are right. It's a old io-testsuite but quite simple. I want to leverage the bonnie test methods for secure storage performance test.